Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# Build: docker build -t hocr-tools-app .
+# Start: docker run -it --rm -v ${PWD}:/usr/src/app hocr-tools-app bash
+# Test: ./test/tsht
+#
+
+FROM python:2
+
+RUN apt-get update && apt-get install -y pdfgrep
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+RUN ["python", "setup.py", "install"]
```

Comment:
Okay, I will change the form. But I don't think we should smash all RUN commands together, because we don't want to trigger the `pip install` every a file (not necessarily the `requirements.txt`) have changed, see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy